components/prism-bsl.js

```diff
@@ -0,0 +1,57 @@
+// 1C:Enterprise
+// https://github.com/Diversus23/
+// todo: Добавить функции (Add function).
+
+Prism.languages.bsl = {
+	'comment': /\/\/.*/,
+	'string': [
+		// strings
+		{
+			pattern: /"(?:[^"]|"")*"(?!")/,
+			greedy: true
+		},
+		// date & time
+		{
+			pattern: /\'(?:\\?.)*?\'/,
```

Comment:
This pattern will backtrack exponentially for the string `'\\\\\\\\\\`.

```suggestion
			pattern: /'(?:[^'\r\n\\]|\\.)*'/,
```